org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/junit/JUnitFragment.xtend

```diff
@@ -25,33 +25,38 @@ import static extension org.eclipse.xtext.GrammarUtil.*
  * @since 2.14
  */
 class JUnitFragment extends AbstractStubGeneratingFragment {
-	
+
 	@Inject extension XtextGeneratorNaming
 	@Inject FileAccessFactory fileAccessFactory
-	
+
 	@Accessors(PUBLIC_SETTER)
 	boolean useDeprecatedClasses
-	
+
 	@Accessors(PUBLIC_SETTER)
 	boolean skipXbaseTestingPackage
-	
+
 	JUnitVersion junitVersion = JUnitVersion.JUNIT_4
 	
-	def void setJunitVersion (String version) {
-		junitVersion = JUnitVersion.fromString(version)
+	
+	new() {
+		generateXtendStub = true
```

Comment:
A small comment why this was done could be helpful since we don't have decent test coverage

Reply:
Done.